messages_appendix.cddl

```diff
@@ -75,23 +75,24 @@ auth-initiation-token = (
   ? 0 : text ; token
 )
 
-; type key 1002
-auth-spake2-need-psk = {
-  auth-initiation-token
-}
+auth-spake2-psk-status = &(
+ psk-needs-presentation: 0
+ psk-shown: 1
+ psk-input: 2
+)
 
-; type key 1003
+; type key 1005
```

Comment:
Should the order of these types be changed now that the type keys are not organized by value?

Reply:
Done